packages/golden_toolkit/lib/src/device_builder.dart

```diff
@@ -176,10 +176,19 @@ class _DeviceScenarioWidget extends StatelessWidget {
       device.size.width + (_horizontalScenarioPadding * 2) + (_borderWidth * 2),
       device.size.height + 24 + (_borderWidth * 2));
 
-  Widget get _sizedWidget => Container(
-        width: device.size.width,
-        height: device.size.height,
-        child: widget,
+  Widget get _sizedWidget => MediaQuery(
```

Comment:
On the surface this looks fine to me. I think a test should be added to demonstrate the desired behavior though so we can prevent regressions

Reply:
True :) will add one in a few hours :D

Reply:
@coreysprague done :)

Reply:
Any update? @coreysprague